examples/faster_rcnn/README.md

```diff
@@ -2,11 +2,25 @@
 
 ### Performance
 
-| Training Setting | Evaluation | Reference Implementation | ChainerCV |
+| Training Setting | Evaluation | Reference | ChainerCV |
 |:-:|:-:|:-:|:-:|
 | VOC 2007 trainval | VOC 2007 test|  69.9 mAP [1] | 70.5 mAP |
 
 
+### Speed
+
+Speed of the Caffe implementation and ChainerCV implementation are almost same in the test mode.
```

Comment:
I am not sure about that.
Since we are talking of the `speed of A and B` (i.e. shorthand for `speed of A and speed of B`), shouldn't they be plural?
I looked it up on the internet :smiling_imp:  [related](http://www.examveda.com/the-speed-of-a-and-b-are-in-the-ratio-34-a-takes-20-minutes-more-than-b-to-reach-a-destination-time-in-which-a-reach-the-destination-386/).

By the way, it seems that it should be `the speed`.